articles/native-platforms/ios-swift/00-introduction.md

```diff
@@ -1,34 +1,176 @@
 ---
 title: Introduction
-description: Brief introduction to the iOS-Swift tutorials.
+description: Brief introduction to the iOS-Swift tutorials. First steps required to follow any of the tutorials.
 ---
 
 This is the very beginning of a simple, practical and multi-step quickstart that will guide you through managing authentication in your iOS apps with Auth0.
 
-::: panel-info System Requirements
-This tutorial and seed project have been tested with the following:
 
-* CocoaPods 1.0.0
-* XCode 7.3 (7D175)
-* Simulator - iPhone 6 - iOS 9.3 (13E230)
-  :::
+
+## Seed Project
+
+There is a [seed project](https://github.com/auth0-samples/auth0-ios-swift-sample/tree/master/00-Starter-Seed) containing an empty project with only the required [dependencies](#dependencies) installed. 
 
 <%= include('../../_includes/_package', {
-  pkgRepo: 'native-mobile-samples',
+  githubUrl: 'https://github.com/auth0-samples'
+  pkgRepo: 'auth0-ios-swift-sample',
   pkgBranch: 'master',
-  pkgPath: 'iOS/basic-sample-swift',
-  pkgFilePath: 'iOS/basic-sample-swift/SwiftSample/Info.plist',
-  pkgType: 'replace'
+  pkgPath: '00-Starter-Seed',
+  pkgFilePath: null,
+  pkgType: 'none'
 }) %>
 
-### Before Starting
+Or… Fork it from GitHub:
+
+<%= include('../../_includes/_github', { link: 'https://github.com/auth0-samples/auth0-ios-swift-sample/tree/master/00-Starter-Seed', }) %>
+
```

Comment:
@pablolvillar These two includes should be merged into one. For example, 01-Starter-Seed should be:
```
<%= include('../../_includes/_package', {
  githubUrl: 'https://github.com/auth0-samples/auth0-ios-swift-sample/tree/master/00-Starter-Seed',
  pkgRepo: 'auth0-ios-swift-sample',
  pkgBranch: 'master',
  pkgPath: '00-Starter-Seed',
  pkgFilePath: null,
  pkgType: 'none'
}) %>
```
Please apply this change to all docs.